### PR TITLE
Edit/readme

### PR DESCRIPTION
**Pull request by eclectic-coding**
Added Quick install section to README.md in response to #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,15 @@ To build on the [Using Contentful](https://github.com/gatsbyjs/gatsby/tree/maste
 Please take a look at the [issue queue](https://github.com/mccrodp/gatsby-contentful-i18n) to help out and feel free to submit PRs :)
 
 See [DEMO](https://gatsby-starter-contentful-i18n.netlify.com/).
+
+## Quick Start
+First, create a new Gatsby site from the starter:
+
+```$ gatsby new my-fun-site https://github.com/cwlsn/gatsby-simple-contentful-starter```
```
Comment (mccrodp) [inline README.md]: Doesn't look like this points to the starter repo URL.

File: README.md [hunk 2]
```diff
@@ -8,3 +8,15 @@ To build on the [Using Contentful](https://github.com/gatsbyjs/gatsby/tree/maste
...
+
+This starter will try to talk to a Contentful Model named homePage.
```
Comment (mccrodp) [inline README.md]: Not sure what the purpose of this line is and I can't remember there being a model name "homePage" anywhere.

**Review by mccrodp — CHANGES_REQUESTED** [review]
Thanks for this, some changes requested.

File: README.md (hunk 2 above)
Comment (eclectic-coding) [reply README.md]: Edits have been made. Thanks.

**Review by mccrodp — APPROVED** [review]
Thanks!

**Pull request merged**